com.lettier.movie-monad.json

```diff
@@ -0,0 +1,1353 @@
+{
+  "app-id": "com.lettier.movie-monad",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "writable-sdk": true,
```

Comment:
This shouldn't be here, and without it you will see my point about `/app` vs `/usr` probably.

Reply:
It follows the provided [example](https://github.com/flathub/org.freedesktop.Bustle/blob/master/org.freedesktop.Bustle.json#L6) as [given](https://github.com/flathub/flathub/pull/335#discussion_r180692889) by @nedrichards. 

I am aware that without it, `/usr` is read only.

:+1:

Reply:
Try installing things into `/app`. If that fails then this workaround is fine but it shouldn't be relied upon.